.github/workflows/after-pullrequest.yml

```diff
@@ -7,7 +7,7 @@ name: "Test Reporter"
 on:
   workflow_run:
     workflows:
-      - "On Pull Request"
+      - "On Pull Request - PowerShell"
```

Comment:
Good catch. Originally I just had "On Pull Request" for the other workflow, and this file was correct. Then I realized that there's another workflow that runs on pull request, so I differentiated by adding " - PowerShell". But that broke the integration with this file.